.github/workflows/testing.yml

```diff
@@ -2,7 +2,6 @@ name: Build and Test
 
 on:
   pull_request:
-  push:
```

Comment:
Does this mean that we no longer run the tests on 'main', only on pull requests?

Reply:
Right, added `main` back.